src/tests/fableconfig.json

```diff
@@ -13,7 +13,7 @@
             "projFile": ["Main/project.fsx"],
             "ecma": "es2015",
             "module": "commonjs",
-            "coreLib": "../../build/fable-core/umd",
+            "coreLib": "../../build/fable-core",
```

Comment:
Yes, it's about location, when fable-core is compiled it goes to /build/fable-core/, not /build/fable-core/umd.